Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM skewled/bulwark-base:6.0.0
```

Comment:
@skewled Please setup the base image as well in the repo or a repo of its own and push it up to Docker hub (https://hub.docker.com/u/softramsdocker)

Reply:
Will add to the bulwark repo in a new folder and have the group docker hub access added and PR updated.